Doc/library/sqlite3.rst

```diff
@@ -98,6 +71,32 @@ This example uses the iterator form::
    ('2006-04-05', 'BUY', 'MSFT', 1000, 72.0)
 
 
+.. _sqlite3-placeholders:
+
+Usually your SQL operations will need to use values from Python variables.  You
+shouldn't assemble your query using Python's string operations because doing so
+is insecure; it makes your program vulnerable to an SQL injection attack
+(see https://xkcd.com/327/ for a humorous example of what can go wrong)::
```

Comment:
Nitpick: Since we are here, maybe replace the direct link with something like:

````
[...] `xkcd comic <https://xkcd.com/327/`>`_ [...]
````